extensions/SemanticMediaWiki/DefaultSettings.php

```diff
@@ -155,7 +155,7 @@
 
 ###
 # If you already have custom namespaces on your site, insert
-#    $smwgNamespaceIndex = ???;
+#    $smwDefaults['smwgNamespaceIndex'] = ???;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)